install/infra/single-cluster/aws/README.md

```diff
@@ -0,0 +1,244 @@
+# Terraform setup for AWS Single-cluster Gitpod reference architecture
```

Comment:
This is _excellent_ documentation! Thinking longer term, how can we link up this documentation to the reference architecture docs so this gets a good level of visibility?

No action needed, resolve when you've seen this.

Reply:
Thanks for pointing this out! We have an issue open for this here https://github.com/gitpod-io/gitpod/issues/11281